### ISSUE TITLE
[Bug]: Unable to resolve service for type 'System.Net.Http.HttpClient' while attempting to activate 'MudBlazor.Extensions.Services.MudExFileService'

### ISSUE THREAD

**Issue by jaypalexa** [fgilde/MudBlazor.Extensions#49]
### Contact Details

jpalexa@duck.com

### What happened?

After upgrading to 1.7.68, we are now seeing the following exception at application startup.  (When we roll back to 1.7.67, we do not see this exception).

```
System.AggregateException
  HResult=0x80131500
  Message=Some services are not able to be constructed (Error while validating the service descriptor 'ServiceType: MudBlazor.Extensions.Services.MudExFileService Lifetime: Transient ImplementationType: MudBlazor.Extensions.Services.MudExFileService': Unable to resolve service for type 'System.Net.Http.HttpClient' while attempting to activate 'MudBlazor.Extensions.Services.MudExFileService'.) (Error while validating the service descriptor 'ServiceType: MudBlazor.Extensions.Services.MudExFileService Lifetime: Transient ImplementationType: MudBlazor.Extensions.Services.MudExFileService': Unable to resolve service for type 'System.Net.Http.HttpClient' while attempting to activate 'MudBlazor.Extensions.Services.MudExFileService'.)
  Source=Microsoft.Extensions.DependencyInjection
  StackTrace:
   at Microsoft.Extensions.DependencyInjection.ServiceProvider..ctor(ICollection`1 serviceDescriptors, ServiceProviderOptions options)
   at Microsoft.Extensions.DependencyInjection.ServiceCollectionContainerBuilderExtensions.BuildServiceProvider(IServiceCollection services, ServiceProviderOptions options)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in C:\temp\TestApp3\Program.cs:line 11

  This exception was originally thrown at this call stack:
    [External Code]

Inner Exception 1:
InvalidOperationException: Error while validating the service descriptor 'ServiceType: MudBlazor.Extensions.Services.MudExFileService Lifetime: Transient ImplementationType: MudBlazor.Extensions.Services.MudExFileService': Unable to resolve service for type 'System.Net.Http.HttpClient' while attempting to activate 'MudBlazor.Extensions.Services.MudExFileService'.

Inner Exception 2:
InvalidOperationException: Unable to resolve service for type 'System.Net.Http.HttpClient' while attempting to activate 'MudBlazor.Extensions.Services.MudExFileService'.
```
```
using MudBlazor.Extensions;

var builder = WebApplication.CreateBuilder(args);
builder.Services.AddHttpContextAccessor();
builder.Services.AddRazorPages();
builder.Services.AddServerSideBlazor();

builder.Services.AddMudExtensions();
builder.Services.AddMudServicesWithExtensions();

var app = builder.Build();
app.UseDeveloperExceptionPage();
app.UseHttpsRedirection();
app.UseStaticFiles();
app.UseRouting();
app.MapBlazorHub();
app.MapFallbackToPage("/_Host");

app.Run();
```

### Expected Behavior

No exception

### Screenshots

![Screenshot_2](https://github.com/fgilde/MudBlazor.Extensions/assets/4446336/32ff2caa-e9da-47c0-84d1-a99bf77da661)
![Screenshot_1](https://github.com/fgilde/MudBlazor.Extensions/assets/4446336/0ef80857-993e-49e5-9154-252c7a97f795)

### What application type are you referring to?

ServerRendered

### MudBlazor.Extension Version

1.7.68

### MudBlazor Version

6.10.0

### What browser are you using?

Chrome

### Sample Solution

[TestApp.zip](https://github.com/fgilde/MudBlazor.Extensions/files/12685831/TestApp.zip)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fgilde** [comment]
You're right, I'm so sorry my mistake. Will change it asap if you need a quickfix you can just register one in your Startup

`builder.Services.AddScoped(sp => new HttpClient { BaseAddress = new Uri(builder.HostEnvironment.BaseAddress) }); `

**Comment by fgilde** [comment]
Solved in 1.7.69
